src/orca-jedi/increment/Increment.cc

```diff
@@ -38,14 +42,19 @@ namespace orcamodel {
 Increment::Increment(const Geometry & geom,
                      const oops::Variables & vars,
                      const util::DateTime & time)
-  : geom_(new Geometry(geom)), vars_(), time_(time),
+  : geom_(new Geometry(geom)), vars_(vars), time_(time),
     incrementFields_()
 {
```

Comment:
I think this class wouldn't work with the fields distributed across more than one processor (the norm calculation for example would be unique to each processor). I think it is good to throw an exception in the increment constructor if there is more than 1 mpi process. Something like the following (untested):
```suggestion
{
  if (geom_->.getComm().size() != 1) {
    throw eckit::NotImplemented("orcamodel::Increment::Increment: Cannot construct an Increment with more than one MPI process.");
  }
```